uwblueprint/sistema
# Style login page

## Notion ticket link

[Login Page](https://www.notion.so/uwblueprintexecs/Login-Page-46b6b3b32e8b41ac821a93fa91cdc9d8)



## Implementation description
- Add Sistema (colour) logo and create `SistemaLogoColour` component
- Add "Only sign in with a Sistema email." text to `index.tsx`
- Style `SignInButton` component
- Remove `SignOutButton` and `session` from `index.tsx`



## Steps to test

1. Visit http://localhost:3000/
2. Confirm "Sign in with Google" button functionality
3. Use Developer Tools to experiment with UI on different screen layouts

Desktop Browser UI:
![Screenshot 2024-10-08 221117](https://github.com/user-attachments/assets/c4b46b8e-eabc-4633-a77d-8744b6f3546b)

(Dev Tool) Tablet UI:
![Screenshot 2024-10-08 222420](https://github.com/user-attachments/assets/cc0a537e-1098-4a0a-a3a4-2b080c4fcfbc)

(Dev Tool) iPhone SE UI:
![Screenshot 2024-10-08 222501](https://github.com/user-attachments/assets/950f604e-5b43-4f12-9101-864b6f1d9a5b)



## What should reviewers focus on?
- UI alignment with mock
![screenshot_2024-10-01_at_9 21 19___pm](https://github.com/user-attachments/assets/60a98087-a4ab-43c1-baea-1a474dfe818c)

- UI transferability (different screen sizes, orientations, etc.)
- General code style/best practices

## Checklist
